tools/docker_image_manager.sh

```diff
@@ -48,7 +48,7 @@ while getopts ":a:r:c:g:hf" option; do
         esac
     done
 
-    if [[ ! "$action" =~ ^(build|publish|save|all)$ ]]; then
+    if [[ ! "$action" =~ ^(build|download|publish|save|all)$ ]]; then
```

Comment:
Also add `download` action in the usage function

Reply:
fixed.